### ISSUE TITLE
Error when using passthrough

### ISSUE THREAD

**Issue by andrew-paterson** [miragejs/mirage-msw#20]
I get the following error if I add passthrough to my server config.

For context this is on an Ember 5.11 app, with `ember-mirage`. If passthrough is removed, everything works great.

```TypeError: import_route_recognizer.default is not a constructor
    at eval (webpack://__ember_auto_import__/./node_modules/.pnpm/mirage-msw@0.2.2/node_modules/mirage-msw/dist/msw-config.cjs?:61:24)
    at Array.forEach (<anonymous>)
    at createPassthroughRegistry (webpack://__ember_auto_import__/./node_modules/.pnpm/mirage-msw@0.2.2/node_modules/mirage-msw/dist/msw-config.cjs?:60:17)
    at PassthroughRegistry.add (webpack://__ember_auto_import__/./node_modules/.pnpm/mirage-msw@0.2.2/node_modules/mirage-msw/dist/msw-config.cjs?:86:33)
    at eval (webpack://__ember_auto_import__/./node_modules/.pnpm/mirage-msw@0.2.2/node_modules/mirage-msw/dist/msw-config.cjs?:381:27)
    at Array.forEach (<anonymous>)
    at MswConfig.passthrough (webpack://__ember_auto_import__/./node_modules/.pnpm/mirage-msw@0.2.2/node_modules/mirage-msw/dist/msw-config.cjs?:376:11)
    at Server.passthrough (webpack://__ember_auto_import__/./node_modules/.pnpm/miragejs@0.2.0-alpha.3/node_modules/miragejs/dist/mirage.cjs?:6463:35)
    at Server.routes (http://localhost:6202/assets/ember-5-11-0-app.js:755:14)
    at Server.loadConfig (webpack://__ember_auto_import__/./node_modules/.pnpm/miragejs@0.2.0-alpha.3/node_modules/miragejs/dist/mirage.cjs?:6457:13)```

**Comment by IanVS** [comment]
Can you share the code you are using or better yet a minimal reproduction?

**Comment by andrew-paterson** [comment]
Sure thing: https://github.com/andrew-paterson/mirage-msw-passthrough

I've managed to work out that Ember is building the app using the `require` export in `package.json`, even though mirage-msw is included via an `import` statement. Thus, the browser is trying to use the code from `/dist/msw-config.cjs`. I'm not sure why this is, but I've asked about it [here](https://discuss.emberjs.com/t/how-to-alias-exports-import-in-package-json-with-ember-auto-import/21387). 

I created and tested a non Ember app, using a basic webpack config, and that bundled the assets using the correct file (`/dist/msw-config.js`).

As another test, I also changed the require export in package.json to `/dist/msw-config.js` for the Ember app and then the error went away.

**Comment by IanVS** [comment]
@bgantzler you're an ember user, right?  Do you have any insights here?

**Comment by cah-brian-gantzler** [comment]
Yes, I am an ember user, but am not using the msw. This seems to be in the build process. based on the above statement that changing the extenstion from `.cjs` to `.js` made the error go away it is outside my understanding of how things work. Hoping Ed may have some insight since it appears to be an auto import issue.

**Comment by andrew-paterson** [comment]
After moving from classic Ember builds to Embroider, I can confirm that the error no longer occurs, with the correct entry point being used. I'm happy to close this issue, as this is clearly more to do with Ember than with this project.
